src/nci_util.c

```diff
@@ -159,10 +159,47 @@ nci_parse_mode_param(
         }
         GDEBUG("Failed to parse parameters for NFC-B poll mode");
         return NULL;
-    default:
-        GDEBUG("Unhandled activation mode %d", mode);
+    case NCI_MODE_ACTIVE_POLL_F:
+    case NCI_MODE_PASSIVE_POLL_F:
+        /*
+         * NFCForum-TS-NCI-1.0
+         * Table 58: Specific Parameters for NFC-F Poll Mode
+         *
+         * +=========================================================+
+         * | Offset | Size | Description                             |
+         * +=========================================================+
+         * | 0      | 1    | Bit Rate (1 = 212 kbps, 2 = 424 kbps)   |
+         * | 1      | 1    | SENSB_REF Response Length n (16 or 18)  |
+         * | 2      | n    | Bytes 2-17 of SENSF_RES                 |
+         * +=========================================================+
+         */
+        if (len > 1 && bytes[1] >= 8 && (bytes[1] + 2) <= len) {
```

Comment:
Random thought: n>=8 tests that about to be accessed amount is available. Should there be also record sanity check vs sizes 16/18? maybe for diagnostic purposes only?

Reply:
Meh, not worth the trouble IMO. I'm only using first 8 bytes of this SENSF_RES thing, anything other than 16 or 18 would be a violation of NCI 1.0 spec, but it says "Other values are RFU" - who knows maybe some other values are already in use by a newer spec that I don't have.